browsers/node16.14.2-slim-chrome103-ff99/global-profile.sh

```diff
@@ -0,0 +1,11 @@
+if [ $(uname -p) = "arm" ]; then
+    printf "\e[31m" # red
+    echo "Warning: You are using the beta Arm build of a cypress/browsers or cypress/included image."
+    echo
+    echo "On Arm, non-Electron browsers are not available, because browser vendors are not yet building for Linux arm64."
+    echo "You must use the built in Electron browser (--browser electron) to run Cypress or find and install unofficial Arm binary builds."
+    echo
+    echo "More details and links to upstream issues for Chrome, Firefox, and Edge can be found at Cypress's issue tracker:"
```

Comment:
Do we have a standard usage of either ```Cypress'``` or ```Cypress's```? For a non-plural possessive noun ending in an ```s```, it looks like either [is correct](https://www.grammarly.com/blog/possessive-case/). Seeing ```Cypress's``` here looked weird to me, though. Tossing it out for consideration.

Reply:
LOL every time I've seen `Cypress'` I've assumed it was a typo. I don't know if there is an official branding guideline for this, we seem to use both internally from a quick search

Reply:
I'd expect `Cypress'` as well